app/models/ability.rb

```diff
@@ -5,16 +5,22 @@ class Ability
   self.ability_logic += [:everyone_can_create_curation_concerns]
 
   # Define any customized permissions here.
-  def custom_permissions
+  def custom_permissions # rubocop:disable Metrics/CyclomaticComplexity
     # Limits deleting objects to a the admin user
     #
     # if current_user.admin?
     #  can [:destroy], ActiveFedora::Base
     # end
 
-    can [:create], Contribution if current_user
-    can [:create], Batch        if current_user
-    can [:manage], DepositType  if current_user
+    can [:create], Contribution if current_user.groups.include?('registered')
+    can [:manage], Contribution if admin?
+
+    can [:create], Batch        if registered_user?
```

Comment:
Is `registered_user?` different than line 15 `current_user.groups.include?('registered')` ?

Might want to make them the same to avoid future confusion or add an explanation why the groups.include is required in the one case